README.md

```diff
@@ -190,6 +197,26 @@ A. Install it in one of the directories pointed to by
      ${BASH_COMPLETION_COMPLETIONSDIR})
    ```
 
+   From bash-completion-2.12, we are going to search the data directory of
+   `bash-completion` under the installation prefix where the target command is
+   installed.  When one can assume that the version of the target
```

Comment:
I think, in this section, we can describe only the search paths where the package may put the completion scripts. The exact search order can be described elsewhere. I have added it as another Q&A 02926c05, but this might possibly be described in a different section if you think it is more appropriate.